crates/wast/src/ast/expr.rs

```diff
@@ -399,9 +399,9 @@ instructions! {
         ElemDrop(ast::Index<'a>) : [0xfc, 0x0d] : "elem.drop",
         TableInit(TableInit<'a>) : [0xfc, 0x0c] : "table.init",
         TableCopy(TableCopy<'a>) : [0xfc, 0x0e] : "table.copy",
-        TableFill(ast::Index<'a>) : [0xfc, 0x11] : "table.fill",
-        TableSize(ast::Index<'a>) : [0xfc, 0x10] : "table.size",
-        TableGrow(ast::Index<'a>) : [0xfc, 0x0f] : "table.grow",
+        TableFill(TableArg<'a>) : [0xfc, 0x11] : "table.fill",
+        TableSize(TableArg<'a>) : [0xfc, 0x10] : "table.size",
+        TableGrow(TableArg<'a>) : [0xfc, 0x0f] : "table.grow",
```

Comment:
Could some tests be added for this syntax where the immediate is optional?

Reply:
Yup, done!